test/jdk/java/text/Format/NumberFormat/CurrencySymbols.properties

```diff
@@ -1,3 +1,28 @@
+#
+# Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This file is not new, so I'd use `2001` as the year created, which is borrowed from the test body `CurrencyFormat.java`.